### ISSUE TITLE
"preview" and "up" commands always show unchanged resources without an option to disable that

### ISSUE THREAD

**Issue by ekini** [pulumi/actions#463]
## Hello!

- Vote on this issue by adding a 👍 reaction
- To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already)

## Issue details

The actions output is quite spammy if you have many resources in the stack, because all of them are shown in the output.
It would be useful to have an option to match the behaviour to the usual "pulumi preview" command, which only shows the changes.

### Steps to reproduce

1. Use the actions with many resources
2. We've got 200 resources and it's hard to understand what's actually changing.

Expected: compact output
Actual: spammy output

### Extra details

I tried to figure it out myself, and it seems like:
1. Actions use pulumi automation SDK, which itself uses the Pulumi CLI.
2. Pulumi CLI always shows spammy output in non-interactive mode.
3. Interactive mode is determined here https://github.com/pulumi/pulumi/blob/4da051a79966673f58d8ca955d69e72a89a71aec/sdk/go/common/util/cmdutil/console.go#L48
4. My thinking was to try and make it "interactive" with "--yes" flag/parameter passed. Should theoretically work with `PULUMI_DISABLE_CI_DETECTION=true PULUMI_SKIP_CONFIRMATIONS=true pulumi preview` in Actions, right?
5. However, the SDK pipes Pulumi CLI output (checked with https://github.com/pulumi/pulumi/blob/2f433d64b73f270b4d56ccb7b566dc14d0d41d2b/sdk/python/lib/pulumi/automation/_cmd.py#L52 , presumably SDKs for other languages do the same) so it won't even work as per https://github.com/pulumi/pulumi/blob/4da051a79966673f58d8ca955d69e72a89a71aec/sdk/go/common/util/cmdutil/console.go#L62

**Comment by leezen** [comment]
Do either the `diff` (which will affect output) or `comment-on-pr` (which will post to your PR) options provide what you need here?

**Comment by ekini** [comment]
Thank you, the `diff` option did make it better! When I was reading the docs it didn't seem to give me less output with `diff` enabled.

`comment-on-pr` was already enabled, but was still hard to parse through ~200 lines.
Still, would be nice to have an option to have `--show-sames` disabled.

**Comment by isaac-jordan** [comment]
I think there's definitely still room for improvement here on the previews - the content of the "Pulumi report" is hundreds of lines of it mostly saying things like: `  ~ azure-native:network:PrivateZone: (refresh)`

I'm using `refresh: true` because I want to capture if there's been any manual changes to resources that will be affected by `pulumi up`.

<img width="444" alt="Image" src="https://github.com/user-attachments/assets/bf1122d9-32a9-4cdc-b6ef-4811bb9c92cd" />

I'm fine with the full report being available under a dropdown, but I'd like a version that just shows the added/removed resources within the PR comment (even better if it's not within a dropdown - since it's important for reviewers to notice those infrastructure changes).

Perhaps a `suppress-refresh: true` option could help me to reduce refresh output - and also a `comment-on-pr-summary: true` to display add/removed outside of the "Pulumi report" dropdown.
